include/RAJA/policy/openmp/policy.hpp

```diff
@@ -42,10 +60,23 @@ struct For {
 struct NoWait {
 };
 
-template <unsigned int ChunkSize>
-struct Static : std::integral_constant<unsigned int, ChunkSize> {
+static constexpr int default_chunk_size = -1;
+
+struct Auto : private internal::Schedule<omp_sched_auto, default_chunk_size>{
```

Comment:
`Auto`, `Static`, and `Runtime` are new types so tagged dispatch works correctly